cmd/kafkalock/kafkalock.go

```diff
@@ -0,0 +1,154 @@
+package main
+
+import (
+	"bufio"
+	"context"
+	"flag"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/Shopify/sarama"
+	"go.uber.org/zap"
+
+	"github.com/grongor/go-kafka-lock/kafkalock"
+)
+
+var (
+	bootstrapServers   = flag.String("bootstrapServers", "", "List of Kafka bootstrap servers, separated by comma")
+	kafkaVersion       = flag.String("kafkaVersion", "2.2.0", "Version of the Kafka cluster")
+	topic              = flag.String("topic", "", "Topic used for the lock, eg. lock.someAppName")
+	maxHolders         = flag.Int("maxHolders", 1, "Maximum allowed number of lock holders, usually just 1 (exclusive lock)")
+	topicPolicy        = flag.String("topicPolicy", "validateOnly", "Topic validation/creation policy, possible values: noValidation, validateOnly, create, dropAndCreate")
+	noTimeoutsTweaking = flag.Bool("noTimeoutsTweaking", false, "Whether or not to use the recommended timeouts")
+	logKafka           = flag.Bool("logKafka", false, "Enable Kafka logging (logs from sarama library)")
+	debug              = flag.Bool("debug", false, "Enable debug logging")
+	flap               = flag.Bool("flap", false, "Instead of trying to get the lock the app will just spam Lock() calls fast to cause many consumer group rebalances")
```

Comment:
The description is missing info what the flag does. Also would rename to `flapGuard` or sth

Reply:
Tt actually does't and I think you don't understand the use for this :D Look at it again please :) This file is basically meant just for testing purposes and this option activates the "I'm totally fucked up app" mode which will just mess with any other locks on the topic

Reply:
ah kk